pkgs/development/libraries/tachyon/enable-jpeg.patch

```diff
@@ -0,0 +1,23 @@
+--- a/unix/Make-config	2015-01-08 20:45:17.968890865 +0100
++++ b/unix/Make-config	2015-01-08 20:46:16.223075085 +0100
+@@ -108,14 +108,14 @@
+ #   http://www.ijg.org/files/
+ ##########################################################################
+ # Uncomment the following lines to disable JPEG support
+-USEJPEG=
+-JPEGINC=
+-JPEGLIB=
++#USEJPEG=
++#JPEGINC=
++#JPEGLIB=
+ 
+ # Uncomment the following lines to enable JPEG support
+-#USEJPEG= -DUSEJPEG
+-#JPEGINC= -I/usr/local/include
+-#JPEGLIB= -L/usr/local/lib -ljpeg
++USEJPEG= -DUSEJPEG
++JPEGINC= -I/usr/include
++JPEGLIB= -L/usr/lib -ljpeg
```

Comment:
Very good point, changed.

It would probably be even easier to just re-write the `Make-config` file entirely instead of patching it, but patching it has the advantage of breaking when upstream changes that file.

Reply:
I think `-ljpeg` is still needed, though

Reply:
I replaced that by setting environment variables in `preBuild`, which is easier to do conditionally and only requires a single patch.